src/lib.rs

```diff
@@ -188,6 +188,10 @@ pub struct InitOptions {
 pub fn init(options: &InitOptions) -> Result<ZWaveManager,()> {
     let mut ozw_options = try!(options::Options::create("./config/", "", "--SaveConfiguration true --DumpTriggerLevel 0 --ConsoleOutput false"));
 
+    try!(options::Options::add_option_int(&mut options, "PollInterval", 60000)); //enable polling each 60 seconds
+    try!(options::Options::add_option_bool(&mut options, "IntervalBetweenPolls", true));
+    try!(options::Options::add_option_bool(&mut options, "ValidateValueChanges", true));
+    try!(options::Options::add_option_bool(&mut options, "Associate", true));
```

Comment:
do you have an idea about what everything does ?
I know the 2 first but not the last 2.